sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1405,6 +1410,26 @@ public partial class BlobBaseClient
         protected virtual Azure.Storage.Blobs.Specialized.BlobBaseClient WithSnapshotCore(string snapshot) { throw null; }
         public virtual Azure.Storage.Blobs.Specialized.BlobBaseClient WithVersion(string versionId) { throw null; }
     }
+    public partial class BlobDirectoryClient
+    {
+        protected BlobDirectoryClient() { }
+        public BlobDirectoryClient(string connectionString, string blobContainerName, string blobDirectoryPath) { }
+        public BlobDirectoryClient(string connectionString, string blobContainerName, string blobDirectoryPath, Azure.Storage.Blobs.BlobClientOptions options) { }
+        public BlobDirectoryClient(System.Uri blobUri, Azure.AzureSasCredential credential, Azure.Storage.Blobs.BlobClientOptions options = null) { }
+        public BlobDirectoryClient(System.Uri blobUri, Azure.Core.TokenCredential credential, Azure.Storage.Blobs.BlobClientOptions options = null) { }
+        public BlobDirectoryClient(System.Uri blobUri, Azure.Storage.Blobs.BlobClientOptions options = null) { }
+        public BlobDirectoryClient(System.Uri blobUri, Azure.Storage.StorageSharedKeyCredential credential, Azure.Storage.Blobs.BlobClientOptions options = null) { }
+        public virtual System.Uri Uri { get { throw null; } }
+        public Azure.Storage.Blobs.Specialized.BlobDirectoryClient CloneClient() { throw null; }
```

Comment:
should CloneClient be public?

Reply:
I was supposed to get rid of this actually, good catch.